src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -41,8 +42,13 @@ export function AlertsPopover({
   services,
   stateContainer,
   onClose: originalOnClose,
+  isPlainRecord,
 }: AlertsPopoverProps) {
   const dataView = stateContainer.internalState.getState().dataView;
+  const query = stateContainer.appState.getState().query;
+  const dateFields = dataView?.fields.getByType('date');
+  const timeField = dataView?.timeFieldName || dateFields?.[0]?.name;
```

Comment:
Would it be helpful to also prioritize `@timestamp` field?

```suggestion
  const timeField = dataView?.timeFieldName || dataView?.getFieldByName('@timestamp') ? '@timestamp' : dateFields?.[0]?.name;
```

Reply:
It is not needed because ESQL already does it. It creates an adhoc dataview with a @timestamp if it exists otherwise an adhoc with no timefield